src/CapabilityService.WebApi/Enablers/PrometheusHealthCheck/PrometheusHealthProbe.cs

```diff
@@ -0,0 +1,54 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Prometheus;
+
+namespace DFDS.CapabilityService.WebApi.Enablers.PrometheusHealthCheck
+{
+    public static class PrometheusHealthProbe
+    {
+        private const string HealthCheckLabelServiceName = "service";
+        private const string HealthCheckLabelStatusName = "status";
+
+        private static readonly Gauge HealthChecksDuration;
+        private static readonly Gauge HealthChecksResult;
+
+        static PrometheusHealthProbe()
+        {
+            HealthChecksResult = Prometheus.Metrics.CreateGauge("healthcheck",
+                "Shows health check status (status=unhealthy|degraded|healthy) 1 for triggered, otherwise 0", new GaugeConfiguration
```

Comment:
Resx files for features/enablers?

Reply:
Maybe you see something I don't yet. As I see it:

- We don't need localization.
- We will not change the strings depending on environment.
- We are not reusing the strings so they don't need to be strongly typed.

Is there a other compelling reason to use a resource file for the PrometheusHealthProbe?

Reply:
In general I like to do "string interpolation" in my code and leave stand-alone strings for resources files to make the code more readable. All the above are added values that often dont come in to play before the code base grows and requirements mature. :) But maybe I am just damaged by having to refactor a desktop healthcare app to chinese where there was no string resource files. :P